mobile app changes for 8.0

richagi opened the pull request:
Added simplified steps and warnings for using mobile app with version 8.0 or higher

azurecla commented:
Hi __@hgoodrow__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@hgoodrow : Thanks for your contribution to the Azure documentation! The author, @cherylmc, has been notified to review your proposed change.

hgoodrow commented:
Why is it taking 8 days to review a very simple PR?   The documentation is clearly wrong and has confused two of my customers.

cherylmc commented:
Hi @hgoodrow - Thanks for letting us know that there was some confusion over that section of the FAQ. I sent this through to the product team to verify. They did recently update some of the wording in the FAQ and what is presently there is accurate. Please let us know if this is still confusing to your customers. Thanks! Cheryl

omartin2010 commented:
I actually think (@ganesr) from PG could confirm, that that traffic can actually flows over both peerings. If you had Azure Public Services and no need for Microsoft Peering, by only enabling the Public Peering, AD Sync would work over the public peering.

charwen commented:
@hgoodrow Azure AD is advertised on both Microsoft Peering and Azure Public Peering. The FAQ is correct. If customer only intends to run Office 365 or CRM Online on ExpressRoute, she doesn't need to enable Azure public peering.

cherylmc commented:
Thanks for the comments, everyone! @tysonn - can you please close this?

richagi committed:
mobile app changes for 8.0

Added simplified steps and warnings for using mobile app with version 8.0 or higher
